assets/plugin-template/android/build.gradle

```diff
@@ -24,6 +24,9 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        aaptOptions {
+            ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:!CVS:!thumbs.db:!picasa.ini:!*~'
```

Comment:
We can probably exclude svn and cvs. Git is the one true way. 

No idea what scc files are.

Picasa is long dead.